volontulo/templates/common/footer.html

```diff
@@ -8,6 +8,7 @@
                 <li><a href="{% url 'organization_view' 1 %}">Widok organizacji</a></li>
                 <li><a href="{% url 'organization_form' %}">Edycja organizacji</a></li>
                 <li><a href="{% url 'offer_apply' 1 %}">Zgłoś chęć współpracy przy ofercie</a></li>
+                <li><a href="{% url 'contact_form' %}">Kontakt z administratorem</a></li>
```

Comment:
it's weird that this change log refers to an older file version even tho i have changed it and it is merged already right now.